server/controllers/column.js

```diff
@@ -0,0 +1,53 @@
+const asyncHandler = require("express-async-handler");
+const Column = require("../models/Column");
+
+exports.getColumns = asyncHandler(async (req, res) => {
+  const columns = await Column.find();
+
+  return res.json(Columns);
+});
+
+exports.getColumn = asyncHandler(async (req, res) => {
+  const column = await Column.findById(req.params.id);
+
+  if (!column) res.status(404).json({ error: "Column not found" });
+
+  return res.status(200).json(column);
+});
+
+exports.createColumn = asyncHandler(async (req, res) => {
+  const { name } = req.body;
+  const newColumn = await Column.create({
+    name: name,
+  });
+
+  return res.status(200).json({ Column: newColumn });
```

Comment:
JSON responses typically don't have `Pascal` case format. Rename to `{ column: newColumn }`

Reply:
Good catch again. I fixed this in the card controller as well. I should be more careful with replaceAll!